code/datums/skills.dm

```diff
@@ -62,6 +62,8 @@ engineer = 0, construction = 0, leadership = 0, medical = 0, surgery = 0, pilot
 	var/police = SKILL_POLICE_DEFAULT
 	var/powerloader = SKILL_POWERLOADER_DEFAULT
 	var/large_vehicle = SKILL_LARGE_VEHICLE_DEFAULT
+	/// Malus or bonuses to all skills
+	var/global_skill_modificator = 0
```

Comment:
Humans with the same set of skills share the same skillset datum, this'll cause problems. My plan on this was to make a helper like modifyRating below that took a single value and applied it to all skills, then call that through the brain's update.